dpnp/dpnp_iface_types.py

```diff
@@ -140,6 +145,74 @@
 PZERO = numpy.PZERO
 
 
+def finfo(dtype):
+    """
+    Returns machine limits for floating-point data types.
+
+    For full documentation refer to :obj:`numpy.finfo`.
+
+    Parameters
+    ----------
+    dtype : dtype, dpnp_array
+        Floating-point dtype or an array with floating point data type.
+        If complex, the information is about its component data type.
+
+    Returns
+    -------
+    out : finfo_object
+        An object have the following attributes
+        * bits: int
+            number of bits occupied by dtype.
+        * eps: float
+            difference between 1.0 and the next smallest representable
+            real-valued floating-point number larger than 1.0 according
+            to the IEEE-754 standard.
+        * max: float
+            largest representable real-valued number.
+        * min: float
+            smallest representable real-valued number.
+        * smallest_normal: float
+            smallest positive real-valued floating-point number with
+            full precision.
+        * dtype: dtype
+            real-valued floating-point data type.
```

Comment:
actually dpctl supports more properties than described here, could you please complete the list